docs/web3.shh.rst

```diff
@@ -7,9 +7,13 @@ SHH API
 The ``web3.shh`` object exposes methods to interact with the RPC APIs under the
 ``shh_`` namespace.
 
+Web3.py exposes select Whisper methods under the ``web3.geth`` and ``web3.parity`` 
+namespaces to interact with the Whisper RPC API based on which endpoints are 
+supported by each respective client.
+
+
 .. warning:: The Whisper protocol is in flux, with incompatible versions supported
-    by different major clients. So it is not currently included by default in the web3
-    instance.
+    by different major clients.
 
```

Comment:
These docs definitely need to be split up & moved - but I was thinking of doing that in a following pr since it's dependent on #1290 which handles splitting up the geth & parity docs. If you think it belongs on this pr, I can include it here once #1290 gets approved & merged.